CMakeLists.txt

```diff
@@ -427,13 +441,15 @@ target_sources(
           src/marlin_stubs/pause/M603.cpp
           src/marlin_stubs/pause/M701_M702.cpp
           src/marlin_stubs/pause/pause.cpp
+          src/gui/guimain.c
```

Comment:
Are you sure it won't break printer without gui?

Reply:
no. I have tested it